desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -17,17 +17,14 @@ GenericName[ar]=مشغل الفلاش
 GenericName[cs]=Flash přehrávač
 GenericName[es]=Reproductor de Flash
 GenericName[fr]=Lecteur Flash
-GenericName[ja]=Flash Player
+GenericName[ja]=Flashプレイヤー
 GenericName[ko]=플래시 플레이어
 GenericName[nl]=Flash speler
 GenericName[pl]=Odtwarzacz Flash
-GenericName[pt_PT]=Flash Player
-GenericName[ru]=Flash Player
-GenericName[uk]=Flash Player
```

Comment:
There's no point to removing these, but it doesn't matter either, as they are the same as the source string.

Reply:
They will be marked as not translated on Crowdin so that people can translate them properly

Reply:
I see, okay!